### PR TITLE
Formatting done

### PR DESCRIPTION
**Pull request by JasmineBeulah**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:
This PR updates the document
-Pint 1
-Point 2

**Which issue(s) this PR fixes**:

Fixes #23 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # contrib-lab01
 This repo is for new contributors trial and learning. This is to support the SODA community members for their learning.
+This is a test Jasmine
```
Comment (PravinRanjan10) [inline README.md:3]: Please add full stop at the end.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+Change
```
Comment (PravinRanjan10) [inline README.md:4]: Change? Could you please elaborate more.

**Review by PravinRanjan10 — CHANGES_REQUESTED** [review]
Please fix the review comments.

File: README.md (hunk 1 above)
Comment (DIVYANUNU) [inline README.md:3]: ADD  . at the end

File: README.md (hunk 2 above)
Comment (DIVYANUNU) [inline README.md:4]: explain pls

File: README.md (hunk 1 above)
Comment (JasmineBeulah) [reply README.md:3]: Done

**Review by DIVYANUNU — CHANGES_REQUESTED** [review]

**Pull request merged**